azure-pipelines.yml

```diff
@@ -34,25 +34,42 @@ variables:
   ${{ if startsWith(variables['Build.SourceBranch'], 'refs/pull/') }}:
     branchName: $[ replace(variables['System.PullRequest.SourceBranch'], 'refs/heads/', '') ]
 
+# Some steps here scaffold jss apps and check if they build and lint correctly
+# build apps step currently excluded: RAV apps have issues with yarn build, nextjs has some with all builds without Sitecore connection
+# using --since for learna commands could be nice but it introduces a lot of branching execution paths that need to be covered
+# TODO: add step for lerna run build --scope sample-* when RAV and nextjs yarn build issues are resolved
```

Comment:
Since we are not doing build now, please add a follow-up work item so we remember to do this in the future :)

Reply:
Yup, all done already :)